src/data/links.ts

```diff
@@ -2,9 +2,10 @@ export const links = {
 	discord: "files",
 	twitter: "FilesForWindows",
 	storeId: "9nghp3dx8hdx",
+	sideloadLink: "https://filescommunity.blob.core.windows.net/files/Files.Package.appinstaller",
```

Comment:
```suggestion
	sideloadLink: "https://filescommunity.blob.core.windows.net/files/stable/Files.Package.appinstaller",
```
I'm also creating a preview version "https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller"